### DISCUSSION TITLE
requested resource wasnt found for users table

### DISCUSSION THREAD

**Discussion by thal-sh** [pocketbase/pocketbase#4968] [Q&A] [answered]
api/collections/users/records/lyr3vh8jx7wgpg9
this is my url, getting a specific user shows "The requested resource wasn't found.", also trying to update the user causes that, tbh i spent on hour on this and i have no idea what to show, the url is correct i can do updates and whatever with other collections, but not the user collection for some reason. 
```javascript
 async function rejectOrder(id: any, user: any, price: any) {
    try {
      await pb.collection("transactions").update(id, { status: "rejected" });
        const data = {
        'balance+': price,
        }
      const userUpdate = await pb
        .collection("users")
        .update(user, data);
    } catch (error) {
      console.error("Error updating user:", error);
    }
  }
```

**Comment by ganigeorgiev** [comment, marked as answer]
Make sure that your client-request satisfies the Update API rule of the `users` collection. From the [docs](https://pocketbase.io/docs/api-rules-and-filters/#api-rules):

> PocketBase API Rules act also as records filter!
> ...
> Because of the above, the API will return 200 empty items response in case a request doesn't satisfy a listRule, 400 for unsatisfied createRule and **404 for unsatisfied viewRule, updateRule and deleteRule**.


If that's not the case, please elaborate a little bit more how do you initialize the SDK, are you authenticated and what are the API rules of the collection you are updating (_you can export it from the Admin UI > Settings > Export collections_).
